site/sst-env.d.ts

```diff
@@ -1 +1,2 @@
+// eslint-disable-next-line @typescript-eslint/triple-slash-reference
 /// <reference path="../.sst/types/index.ts" />
```

Comment:
```suggestion
// <reference path="../.sst/types/index.ts" />
```

Reply:
As per above comment this file is removed

Reply:
why does this need to be commented in the first place?